packages/calypso-e2e/src/lib/pages/signup/signup-pick-plan-page.ts

```diff
@@ -28,10 +28,7 @@ export class SignupPickPlanPage {
 	 * @returns {Promise<SiteDetails>} Details of the newly created site.
 	 */
 	async selectPlan( name: Plans ): Promise< NewSiteResponse > {
-		await Promise.all( [
-			this.page.waitForURL( /.*start\/plans.*/ ),
-			this.page.waitForLoadState(),
-		] );
+		await Promise.all( [ this.page.waitForLoadState() ] );
```

Comment:
Yes, it's intentional because the plans page for the `with-theme` is different (`/start/with-theme/plans-theme-preselected?theme=theme-name`).

Good point. There's no reason to keep it in the Promise.all.